docs/index.rst

```diff
@@ -112,7 +112,7 @@ The OPC UA Advanced tab allows advanced configuration parameters to be set.
     The path does not include the *Objects* folder or the subscribed Node.
   - **Full OPC UA Path meta data name**: The data point name to use when adding the full OPC UA path to every reading. Default is *OPCUAPath*.
   - **Debug Trace File**: Enable the S2OPCUA OPCUA Toolkit trace file for debugging. If enabled, log files will appear in the directory */usr/local/fledge/data/logs*.
-  - **MonitoredItem block size**: The number of MonitoredItem to be added on to the OPCUA server in single call.
+  - **MonitoredItem block size**: The number of MonitoredItems to be registered with the OPC/UA server in single call to the S2OPCUA OPCUA Toolkit. MonitoredItem is the OPC/UA term for a single data stream registered for data updates from an OPC/UA server. MonitoredItem block size can be minimum 1 and maximum the 32-bit integer value. Default value will work fine for most of the cases. Use lower value if following error **Failed to add <N> Monitored Items** appears in the system logs.
```

Comment:
I would suggest something more like "The number of items passed in single call to the S2OPCUA OPCUA Toolkit when requesting to monitor data change events in the OPC/UA server. The default value will work in most of the cases. If the error **Failed to add <N> Monitored Items** is written to the logs then try reducing the number of items sent in each call until this error stops occurring. Reducing the number too far will impact one the performance, increasing the time it takes to setup the connection with the server and get the first data back from the server. Using very large values for this call will put extra stress on the OPC/UA server and also increase the memory footprint of the plugin. The minimum value of this is 1, the default is 100."

Reply:
I like this text. One question: there is a sentence that begins "_Reducing the number too far will impact one the performance..._" The word "_one_" doesn't make sense here. Was this a typo of another word? You could just leave it out.